src/sentry/web/urls.py

```diff
@@ -127,14 +127,14 @@
         ReleaseWebhookView.as_view(),
         name="sentry-release-hook",
     ),
-    url(r"^api/embed/error-page/$", ErrorPageEmbedView.as_view(), name="sentry-error-page-embed"),
+    url(r"^api/embed/error-page/?$", ErrorPageEmbedView.as_view(), name="sentry-error-page-embed"),
```

Comment:
My understanding of how the trailing slash logic has worked is that we explicitly do not match on optional trailing slash, and will automatically redirect with the slash if it's missing right?